hack/render-sync.sh

```diff
@@ -30,6 +32,6 @@ function rendersync() {
   rm -r "${ARTIFACT_DIR}"
 }
 
-rendersync manual-cluster/performance base/performance default
-rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap/no-mcp
-rendersync bootstrap-cluster/performance pinned-cluster/default bootstrap-cluster/extra-mcp bootstrap/extra-mcp
\ No newline at end of file
+rendersync manual-cluster/performance base/performance default label-name
```

Comment:
I think mixing directories and ownership reference arguments is confusing a bit. The traditional unix way would be something like:

`rendersync --owner-ref label-name -- manual-cluster/performance base/performance default`

In other words, extra arguments followed by -- separator and then the rendering arguments.

The code to implement this is a bit more complicated than what you have here, but I believe it would make the code less error prone.

Reply:
This would be the best option indeed. I was about to settle for a little less, but if we can get this far that would be great.

Reply:
The only thing that bothers me with this approach is that we basically implement `render` command line again but in the lesser convenient way which is using bash for parsing the arguments.

Reply:
Maybe we can enhance render command to read its configuration from yaml file something like:
``` yaml
config:
  - inputDir: "foo"
    outputDir: "bar"
    ownerRef: none
 -  inputDir: "foo2"
    outputDir: "bar2"
    ownerRef: k8s
 -  inputDir: "foo2"
    outputDir: "bar2"
    // use default owner-ref
```

Reply:
I think we can implement my proposal like this. Just below line 7 include

```
EXTRA_ARGS=()
if [[ "$1" =~ ^-.* ]]; then
    while (( "x$1" != "x--" )); do
        EXTRA_ARGS+=("$1")
        shift
    done
    shift
fi
```

And then in the execution of render use `"${EXTRA_ARGS[@]}"`

We should make the right value the default one in the render command so we do not have to use this method too often.